packages/panw_cortex_xdr/data_stream/incidents/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,212 @@
+---
+description: Pipeline for Palo Alto XDR Incident API.
+processors:
+  - set:
+      field: ecs.version
+      value: '8.9.0'
+  - set:
+      field: event.kind
+      value: alert
+  - append:
+      field: event.category
+      value: malware
+  - append:
+      field: event.type
+      value: info
+  - rename:
+      field: message
+      target_field: event.original
+  - json:
+      field: event.original
+      target_field: panw_cortex.xdr
+  - drop:
+      if: ctx.panw_cortex?.xdr?.reply?.result_count != null && ctx.panw_cortex?.xdr?.reply?.result_count == 0
```

```suggestion
      if: ctx.panw_cortex?.xdr?.reply?.result_count == 0
```
If the second clause is true, the first must have been, so it can be elided.